src/core_mapfiles/ecl.rs

```diff
@@ -629,7 +629,7 @@ static ECL_08_09: &'static CoreSignatures = &CoreSignatures {
         (Th08, 127, Some(("S", None))),
         (Th08, 128, Some(("S(imm)f(imm)f(imm)f(imm)f(imm)", None))), // Argument 1 is unread
         (Th08, 129, Some(("b(imm)---", None))),
-        (Th08, 130, Some((r#"s(imm;enum="EclSub")--"#, None))),
+        (Th08, 130, Some((r#"s(imm;extend;enum="EclSub")--"#, None))),
```

Comment:
There're quite a few places where the game engines will read a full dword from the signature in order to check for variables but then only read the lower bits of the resulting value. Ideally `extend` could be replaced with a more general purpose attribute applicable to those situations as well since that would eliminate the need for extended padding anyway. Maybe this could be written as `E(imm;range="s")` or something?

Reply:
I was also thinking this should probably be a dword-sized value with an attribute that limits its value size.  However, I really do not want to see abi letters get used inside string literals.